includes/class-wc-user-tracking.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Nosara Tracks for Woo
+ */
+
+require_once( dirname( __FILE__ ) . '/libraries/tracks/class-tracks.php' );
```

Comment:
This file is called `class-wc-user-tracking.php` but there is no class inside... This probably will be a nice place to load all classes from this library and init it properly.